README.md

```diff
@@ -1,135 +1,170 @@
 
-When fine-tuning a `pyannote` segmentation model on a given dataset, be sure to specify this: 
-
-- `dataset_name`: Specify a dataset from the Hub on which to fine-tune your model.  
-- `dataset_config_name`:  If the dataset contains multiple language subsets, select the language ID of the subset you want to train on.
-- `train_split_name`: Specify which dataset split is to be used for training. Default is 'train'. 
-- `eval_split_name`: Specify which dataset split is to be used for validation. Default is 'validation'. 
-
-If the data set doesn't already contain a train and a validation split, you can automatically split it into train-val-test (90-10-10) using: 
-
-- `do_split_on_subset`: Specify the subset of the dataset you want to split into train-val-set.
+The script [`train_segmentation.py`](train_segmentation.py) can be used to pre-process a diarization dataset and subsequently
+fine-tune the `pyannote` segmentation model. In the following example, we fine-tune the segmentation model on the Japanese
+subset of the [CallHome dataset](https://huggingface.co/datasets/diarizers-community/callhome), a conversational dataset
+between native speakers:
 
-```
-python3 train_segmentation.py
+```bash
+python3 train_segmentation.py \
     --dataset_name=diarizers-community/callhome \
-    --dataset_config_name=jpn \ 
-    --do_split_on_subset=data \
+    --dataset_config_name=jpn \
+    --split_on_subset=data \
     --model_name_or_path=pyannote/segmentation-3.0 \
-    --output_dir=diarizers-community/speaker-segmentation-fine-tuned-callhome-jpn \
-    --preprocessing_num_workers=2 \ 
+    --output_dir=./speaker-segmentation-fine-tuned-callhome-jpn \
     --do_train \
-    --do_eval \ 
-    --learning_rate=1e-3 \ 
-    --num_train_epochs=1 \
-    --per_device_train_batch_size=32 \ 
-    --per_device_eval_batch_size=32 \ 
+    --do_eval \
+    --learning_rate=1e-3 \
+    --num_train_epochs=1 \
+    --per_device_train_batch_size=32 \
+    --per_device_eval_batch_size=32 \
     --evaluation_strategy=epoch \
-    --save_strategy=epoch \  
-    --dataloader_num_workers=2 \ 
-    --logging_steps=200 \
+    --save_strategy=epoch \
+    --preprocessing_num_workers=2 \
+    --dataloader_num_workers=2 \
+    --logging_steps=5 \
     --load_best_model_at_end \
     --push_to_hub
 ```
 
-## Inference
+On a single NVIDIA RTX 24GB GPU, training takes approximately 2 minutes and improves the diarization error rate (DER) 
+from 27% to 21%, representing a 22% relative improvement in performance. The final model will be pushed to the Hugging Face
+Hub, for example the checkpoint [diarizers-community/speaker-segmentation-fine-tuned-callhome-jpn](https://huggingface.co/diarizers-community/speaker-segmentation-fine-tuned-callhome-jpn).
 
-Evaluate your model at inference time using: 
+We encourage you to swap the CallHome Japanese dataset for a dataset in your language of choice. The [CallHome dataset](https://huggingface.co/datasets/diarizers-community/callhome)
```

Comment:
Let's really try and push the community to substitute in a dataset in their language of choice